content/blog/2023-02-28-ethereum-classic-is-a-commodity-ethereum-is-a-security/index.md

```diff
@@ -0,0 +1,140 @@
+
+This level of centralization and the lack of the proof of work cryptographic stamps when producing blocks completely eliminate the chain selection point, thus the attributes of free exit and entry, permissionlessness, censorship resistance, and immutability in Ethereum, making it a common enterprise with traditional centralized features and controls.
+
+ETC remains a commodity because it is and will remain Proof of Work (PoW) for the foreseeable future. 
+
+Therefore, there aren’t and will never be enterprises or groups who will control ETC, and mining and owning the coin will always be totally separate activities. 
+
+ETC has no foundation directing the roadmap of the system, and producing or verifying blocks are free, decentralized, and merit based activities.
+
+### 3. A reasonable expectation of profit: 
+
+As said before, although both ETH and ETC may be used purely as units of exchange, they may also be used as a store of value for price appreciation, therefore both qualify as having a reasonable expectation of profit.
+
+### 4. Derived from the efforts of others: 
+
+For the same reasons that Ethereum is now a common enterprise it is also a system where the expectation of the success of the system, and thus its future profits, are derived from the efforts of others. 
+
+These “others” will be perfectly identifiable, as they will be 4 or 5 large staking pools and the Ethereum Foundation, and all together will act as a single enterprise. 
+
+The PoS system is not a free merit system as PoW but a totally collaborative partnership between the pools and the validators, and the validators are essentially the same as the pools because they are their submissive contractors.
+
+In the system, validators divide their tasks on a per block basis where one produces a block, then sends it to the other validators who vote on the block, and then send it to the the rest of the network who must accept them without objection.
+
+Because ETC is PoW and decentralized, miners may pop up or leave whenever they want from anywhere in the world, compete to build blocks in their complete isolation, send them to the rest of the network for verification, and get paid based on their sole merit. No other filters or conditions exist. 
+
+In ETC, the same reasons why it was a commodity before remains today: There is no common enterprise, therefore the future value of the token is just determined by future general adoption by the market, but not from a defined and collaborative group of centralized pool operators or managers as in Ethereum.
+
+## A Historical Perspective of ETC’s and ETH’s Legal Status
+
+![The crowdsale.](./2.png)
+
+From 2014 to 2016 both Ethereum and Ethereum Classic were one project, so we refer to them in this period as ETH/ETC. 
+
+The project was financed with a crowdsale. Based on the Howey test it may be inferred that the ETH/ETC [crowdsale was, indeed, a security](https://twitter.com/WendellMoone/status/1623760982672564226). 
```

Comment:
Line 86: Add comma
```suggestion
The project was financed with a crowdsale. Based on the Howey test, it may be inferred that the ETH/ETC [crowdsale was, indeed, a security](https://twitter.com/WendellMoone/status/1623760982672564226). 
```